bftengine/src/bftengine/PrimitiveTypes.cpp

```diff
@@ -19,5 +19,20 @@ std::string CommitPathToStr(CommitPath path) {
   }
 }
 
+std::string CommitPathToMDCString(CommitPath path) {
+  switch (path) {
+    case CommitPath::NA:
+      return "NA";
+    case CommitPath::OPTIMISTIC_FAST:
+      return "fast";
```

Comment:
You can remove this line, `switch case` is fall-through.

Reply:
I believe you meant for the two "fast" is a row.
sure